src/games/strategy/triplea/ui/screen/IDrawable.java

```diff
@@ -1,5 +1,19 @@
 package games.strategy.triplea.ui.screen;
 
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.TerritoryEffect;
+import games.strategy.engine.data.Unit;
+import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.attachments.TerritoryAttachment;
+import games.strategy.triplea.formatter.MyFormatter;
+import games.strategy.triplea.image.MapImage;
+import games.strategy.triplea.image.TileImageFactory;
+import games.strategy.triplea.ui.IUIContext;
+import games.strategy.triplea.ui.MapData;
+import games.strategy.triplea.util.Stopwatch;
```

Comment:
Thought we had killed off the StopWatch usages. You're right there is something that is being timed with it.